src/main/java/com/codeborne/selenide/TextMatchOptions.java

```diff
@@ -0,0 +1,56 @@
+package com.codeborne.selenide;
+
+import java.util.Map;
+
+import static com.codeborne.selenide.CaseSensitivity.CASE_INSENSITIVE;
+import static com.codeborne.selenide.CaseSensitivity.CASE_SENSITIVE;
+import static com.codeborne.selenide.TextCheck.FULL_TEXT;
+import static com.codeborne.selenide.TextCheck.PARTIAL_TEXT;
+import static com.codeborne.selenide.WhitespaceHandling.IGNORE_WHITESPACES;
+import static com.codeborne.selenide.WhitespaceHandling.PRESERVE_WHITESPACES;
+import static com.codeborne.selenide.WhitespaceHandling.TRIM_WHITESPACES;
+import static java.util.Locale.ROOT;
+
+public record TextMatchOptions(TextCheck textCheck, CaseSensitivity caseSensitivity, WhitespaceHandling whitespaceHandling) {
+  private static final TextMatchOptions fullText = new TextMatchOptions(FULL_TEXT, CASE_SENSITIVE, TRIM_WHITESPACES);
```

Comment:
This variable not reusable across other code, let hide it inside method `fullText()`